packages/cli/src/__tests__/artificial-panic.test.ts

```diff
@@ -48,8 +56,7 @@ describe('artificial-panic introspection', () => {
 
       `),
         expect(e).toMatchObject({
-          area: 'INTROSPECTION_CLI',
-          schemaPath: undefined,
+          area: 'LIFT_CLI',
```

Comment:
If now introspection errors also report `LIFT_CLI`, do we still have enough information in the error reporting tool to quickly place where the error comes from?

Reply:
Yes, once https://github.com/prisma/prisma-private/issues/203 is closed (`prisma@4.10.0`)

Reply:
How does that influence it?

Reply:
Besides the actual database version (which is WIP) and the error area (which doesn't matter), this PR will preserve every panic information as before

Reply:
> and the error area (which doesn't matter)

There is literally a filter by error area in the error report tool.